### PR TITLE
Add color in settings

### PR DESCRIPTION
**Pull request by emagdalenaC2i**

### CODE REVIEW

File: weld.fs.js [hunk 1]
```diff
@@ -227,7 +239,7 @@ export const weld = defineFeature(function(context is Context, id is Id, definit
             setProperty(context, {
                         "entities" : qCreatedBy(id, EntityType.BODY),
                         "propertyType" : PropertyType.APPEARANCE,
-                        "value" : color(0.25)
+                        "value" : color(definition.colorRed, definition.colorGreen, definition.colorBlue, definition.colorAlpha)
```
Comment (mbartlett21) [inline weld.fs.js]:
```suggestion
                        "value" : color(definition.colorRed, definition.colorGreen, definition.colorBlue, 1 - definition.colorTransparency)
```

File: weld.fs.js [hunk 2]
```diff
@@ -196,6 +196,18 @@ export const weld = defineFeature(function(context is Context, id is Id, definit
...
+            isReal(definition.colorRed, { (unitless) : [0.0, 0.25, 1] } as RealBoundSpec);
+
+            annotation { "Name" : "Color Green", "UIHint" : "REMEMBER_PREVIOUS_VALUE" }
+            isReal(definition.colorGreen, { (unitless) : [0.0, 0.25, 1] } as RealBoundSpec);
+
+            annotation { "Name" : "Color Blue", "UIHint" : "REMEMBER_PREVIOUS_VALUE" }
+            isReal(definition.colorBlue, { (unitless) : [0.0, 0.25, 1] } as RealBoundSpec);
+
+            annotation { "Name" : "Transparency", "UIHint" : "REMEMBER_PREVIOUS_VALUE" }
+            isReal(definition.colorAlpha, { (unitless) : [0.0, 1, 1] } as RealBoundSpec);
```
Comment (mbartlett21) [inline weld.fs.js]:
```suggestion
            isReal(definition.colorTransparency, { (unitless) : [0.0, 1, 1] } as RealBoundSpec);
```

**Comment by mbartlett21** [comment]
Could you update the Onshape document and release a version, please?

**Comment by emagdalenaC2i** [comment]
I was going to do it now, but I think you're already there. I was going to ask you for the version number 0.4.10 or if you go up to 0.5.0

**Comment by mbartlett21** [comment]
I have just done [V 0.4.11](https://cad.onshape.com/documents/7c7042b84e1ae5b391af796f/v/939cf5dbe30c73ec2391d25e)

**Pull request merged**